release-note-script/src/main/java/ReleaseNoteCreation.java

```diff
@@ -280,7 +291,8 @@ private String getProjectId() throws Exception {
       BufferedReader br =
           runSubProcessAndGetOutputAsReader(
               format(
-                  "gh project list --owner %s --closed | awk '/%s/ {print}' | awk '/%s/ {print $1}'",
+                  "gh project list --owner %s --closed | awk '/%s/ {print}' | awk '/%s/ {print"
```

Comment:
Just out of curiosity, but why is this line split? The original one is easier to read for me.